### PR TITLE
Added the initial version for the support of rego language

### PR DESCRIPTION
**Pull request by FallenAngel97**
Hi!

Here's a basic support for rego language 
https://www.openpolicyagent.org/docs/latest/policy-language/

### CODE REVIEW

File: queries/rego/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+; highlights.scm
+"import" @include
+"package" @include
+
+
+(reserved_keywords) @keyword
+(comment) @comment
```
Comment (stsewd) [inline queries/rego/highlights.scm:7]: you could also add the comment parser to the injections, like in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/go/injections.scm

**Review by stsewd — APPROVED** [review]

File: queries/rego/highlights.scm (hunk 1 above)
Comment (FallenAngel97) [reply queries/rego/highlights.scm:7]: Done. 
But can you please, help me understand, why that file is needed? 
As I understood, injections are a part of external bindings as C, but there is none

**Review by theHamsta — APPROVED** [review]

**Review by theHamsta — CHANGES_REQUESTED** [review]
You need to run `stylua` at the `parser.lua` file. The other failures of the stylua jobs are unrelated

**Comment by FallenAngel97** [comment]
> You need to run `stylua` at the `parser.lua` file. The other failures of the stylua jobs are unrelated

Done!

**Review by theHamsta — APPROVED** [review]

**Comment by FallenAngel97** [comment]
Why tests have failed? 
It seems, everything succeeded still it prevents from merging...

**Comment by clason** [comment]
The tests are just flaky. Rerunning it now.

File: queries/rego/highlights.scm (hunk 1 above)
Comment (stsewd) [reply queries/rego/highlights.scm:7]: you can check our docs about this at https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#injections. Injections are done from the client using tree-sitter, not from the parser.

**Pull request merged**